### ISSUE TITLE
Missing support for text/vtt transcripts

### ISSUE THREAD

**Issue by curthipster** [RyanHirsch/partytime#32]
Noticed this in the output while using this library:

```
Unexpected transcript type text/vtt
Please open an issue - https://github.com/RyanHirsch/partytime/issues
```

Here's an example: https://podnews.net/rss

```xml
<podcast:transcript url="https://podnews.net/audio/podnews230706.mp3.vtt" type="text/vtt" />
<podcast:transcript url="https://podnews.net/audio/podnews230706.mp3.srt" type="application/srt" />
<podcast:transcript url="https://podnews.net/audio/podnews230706.mp3.srt" type="application/x-subrip" rel="captions" />
```

**Comment by RyanHirsch** [comment]
Sorry for the delay, addressed with version `4.6.1`

**Comment by curthipster** [comment]
@RyanHirsch no worries, thank you!
